packages/discord/src/bot.ts

```
@@ -497,7 +591,15 @@ ${formatAPIError(error)}`;
 		return 'U' + interaction.user.id;
 	}
 
-	private async canUseAuthCommand (interaction: ChatInputCommandInteraction): Promise<boolean> {
+	private getIdFromMessage (message: Message): string {
+		if (message.inGuild()) {
+			return 'G' + message.guildId;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)